features.json

```diff
@@ -78,13 +78,14 @@
 			"logo": "/images/firefox.svg",
 			"version": "77",
 			"features": {
-				"bigInt": "Nightly",
-				"bulkMemory": "javascript.options.shared_memory",
-				"multiValue": "Nightly",
+				"bigInt": "Beta 78",
+				"bulkMemory": "Nightly",
```

Comment:
This is available in Nightly without any flags

Reply:
it seems that it's because `javascript.options.shared_memory` is enabled in Nightly. I'll revert this also